docs/services/vision/classification.md

```diff
@@ -88,7 +88,26 @@ Proceed to [test your classifier](#test-your-classifier).
 
 ## Test your classifier
 
-You can test your classifier with [live camera footage](#live-camera-footage) or [existing images](#existing-images).
+You can test your classifier with [existing images in the Viam app](#existing-images-in-the-cloud), [live camera footage](#live-camera-footage), or [existing images on a computer](#existing-images-on-your-machine).
+
+### Existing images in the cloud
+
+{{< alert title="Note" color="note" >}}
+
+Your classifier must have been trained or uploaded after the September 19, 2023.
+
+{{< /alert >}}
+
+{{<gif webm_src="/services/vision/mug-classifier.webm" mp4_src="/services/vision/mug-classifier.mp4" alt="A classification model run against an image containing a mug." max-width="250px" class="alignright">}}
+
+If you have images stored in the [Viam cloud](/manage/data), you can run your classifier against your images in the app.
```

Comment:
```suggestion
If you have images stored in the [Viam cloud](/manage/data), you can run your classifier against your images in the [Viam app](https://app.viam.com/).
```

Let's link directly here; Step 1 links to a how-to, not to the target UI. So let's have the target UI link here.